native/packages/react-native-bpk-component-navigation-bar/src/BpkNavigationBar.ios.js

```diff
@@ -39,7 +39,10 @@ import BpkNavigationBarTextButtonIOS from './BpkNavigationBarTextButtonIOS';
 import BpkNavigationBarIconButtonIOS from './BpkNavigationBarIconButtonIOS';
 import isIphoneX from './isIphoneX';
 
-const IOS_THEME_ATTRIBUTES = [...THEME_ATTRIBUTES, 'navigationBarShadowColor'];
+const IOS_THEME_ATTRIBUTES = [
+  ...THEME_ATTRIBUTES.filter(x => x !== 'navigationBarStatusBarStyle'), // `navigationBarStatusBarStyle` is optional on iOS
+  'navigationBarShadowColor',
+];
 
 const statusBarPadding = isIphoneX ? 44 : 20;
```

Comment:
unrelated question: is it fine to have magic numbers here?

Reply:
Yes it's confined to only this file